app/src/screen-components/user-news/news-list.tsx

```diff
@@ -252,18 +254,27 @@ const NewsNavList: FC<NewsList> = ({ style, categories, timeRange }) => {
     if (data == undefined) {
       return;
     }
-    const enriched = data.articles.edges.map((item, index) => {
-      const randomTrackNumber = Math.floor(Math.random() * 15 + 1);
-      return {
-        ...item.node,
-        img: "https://picsum.photos/200/140/?id" + item.node.id,
-        url:
-          "https://www.soundhelix.com/examples/mp3/SoundHelix-Song-" +
-          randomTrackNumber +
-          ".mp3",
-        artist: item.node.provider.name,
-      };
-    });
+
+    const enriched = data.articles.edges
+      .filter((item) => {
+        if (item.node != null) {
+          return item;
+        }
+        return null;
+      })
```

Comment:
hmm to je zvláštní, že backend vrací prázdný node, možná by se to mělo spíš ošetřit tam?

Reply:
BE je under development, mozna uz to neni v tento okamzik ani potreba. Jakub vsechno co objevim hned opravuje